commons/commons-model-extended/src/test/kotlin/ebi/ac/uk/extended/model/ExtSubmissionExtensionsTest.kt

```diff
@@ -1,11 +1,23 @@
 package ebi.ac.uk.extended.model
 
+import arrow.core.Either.Companion.left
+import ebi.ac.uk.io.ext.md5
+import ebi.ac.uk.io.ext.size
+import io.github.glytching.junit.extension.folder.TemporaryFolder
+import io.github.glytching.junit.extension.folder.TemporaryFolderExtension
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
 import java.time.OffsetDateTime
 import java.time.ZoneOffset
 
-class ExtSubmissionExtensionsTest {
+@ExtendWith(TemporaryFolderExtension::class)
+class ExtSubmissionExtensionsTest(
+    tempFolder: TemporaryFolder
```

Comment:
this is only needed in  one test so we can declare both variables (files) and folder directly in test case